docs/internals/contributing/index.txt

```diff
@@ -6,59 +6,88 @@ Django is a community that lives on its volunteers. As it keeps growing, we
 always need more people to help others. You can contribute in many ways, either
 on the framework itself or in the wider ecosystem.
 
+Communication channels
+======================
+
+Communication is key - working on Django is being part of a conversation.
+Join it, to become familiar with what we're doing and how we talk about it.
+You'll be able to form relationships with more experienced contributors who
+are there to help guide you towards success.
+
+Join the |django-developers| mailing list and share your ideas for how to
```

Comment:
Just looking at this now as the last of your comments to address and I can't find bullet points for communication channels.

Could you point me in the right direction/give me a string to search the docs for please?

Reply:
Sure! Further down this page, there is a section `Join the Django community` which sort of duplicates what this paragraph is adding.

I guess we could either move that existing section up, or merge this newly added section down into the existing one. What do you think?

Reply:
As much as I'd like to avoid sharing a screenshot of it, I think this illustrates it better than me describing the change.

How does this look, with a new sentence linking to the community page, and then that section brought up detailing the community parts. I think it still justifies it's own heading as well.

<img width="676" alt="Screenshot 2024-01-10 at 00 30 02" src="https://github.com/django/django/assets/1461191/a40f2bf2-f772-4c3d-8718-d7df97d51cdb">

Reply:
Did you push this? :thinking:

I'm not seeing this when using the latest revno of this branch. If you haven't pushed it yet, let me push a commit with the changes I'm proposing (it includes some related cleanup).

Reply:
Yeah, I haven't committed this change. I just shuffled it round to see what you thought & to see if I had the right idea.